components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -51,6 +57,12 @@ main() {
   exit_with_return_code
 }
 
+verify_scans_support_tools() {
```

Comment:
0: When it comes time to add support to other experiments (like Maven exp 2), we will want to move this to a lib script.